src/components/calcite-date-picker/utils.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { getLocaleData, requestCache, translationCache } from "./utils";
+import { getAssetPath } from "@stencil/core";
+
+function setupFetchStub(data: any) {
+  return function fetchStub(): Promise<any> {
+    return new Promise((resolve) => {
+      resolve({
+        json: () => Promise.resolve({ data })
+      });
+    });
+  };
+}
+
+describe("utils", () => {
+  describe("getLocaleData", () => {
+    const fakeData = { fake: "fake data not meant to be checked" };
```

Comment:
Can you move the mock setup in a `beforeEach()` function and the teardown from each test (clearing the locale) in an `afterEach()`?